style.css

```diff
@@ -357,6 +383,29 @@ body {
     gap: 0.75em;
   }
 
+  .popup {
+    position: fixed;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    overflow: auto;
+    background: rgba(0, 0, 0, 0.7);
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-bottom: 39px;
+    padding-top: 60px;
+    z-index: 2;
+    visibility: hidden;
+  }
```

Comment:
Amazing job on the mobile popup! I have **a couple of small suggestions to further improve it and bring it closer to the provided** [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=6Kf7cezM539HxgRb-0):

- [ ] Please double-check that the popup is correctly centered on all devices like 375x667 or 414x896. It seems that the fixed `padding-top` and `padding-bottom` you added might **be causing some alignment issues**. I recommend using percentages or considering **centering it with Flexbox** for a better result. 🔍

- [ ] Additionally, please ensure that you're using **the correct image from the template**. It appears that you're currently using a different one. Please make the necessary adjustments to match the template more closely. 🙋‍♂️

- [ ] **[OPTIONAL]** You could consider adding a bit more example text to your paragraph to make it look closer to the template. 😉

  _Watch the next video to gain a better understanding and see a possible solution for your problem._ 🎬

  [5_19_2023_4-36-34 PM.webm](https://github.com/Kanyorok/Portfolio_Setup/assets/107952724/ce60e799-0399-41ba-9508-9c0306ee421a)